packages/smart-board/src/SmartBoard/ChartToBoard.ts

```diff
@@ -0,0 +1,201 @@
+/* eslint-disable no-underscore-dangle */
+// @ts-ignore
+import * as louvain from 'louvain-algorithm';
+
+import { Chart, InputChart, ChartOrder, ChartGraph, ChartCluster, Link, SmartBoardType } from '../interfaces';
+import { getDegreeImportance, getChartConnection } from '../utils/calculator';
+import { addUuid } from '../utils/addUuid';
+
+/**
+ * ChartToBoard as SmartBoard (the original SB)
+ * Get Dashboard from chart lists
+ * @public
+ */
+export class SmartBoard {
+  private charts: Chart[];
+
+  private _chartGraph: ChartGraph | undefined;
```

Comment:
In my impression, the conclusion of one discussion is that private variables do not need to be prefixed